### PR TITLE
[Batch] Update Batch API

### PR DESCRIPTION
**Pull request by bgklein**

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S168

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -60,6 +60,10 @@ Release History
 * Added support for Backing up OS Disks only
 * Added restore-as-unmanaged-disk parameter to specify unmanaged restore
 
+**Batch**
+* az batch pool create: Added support to specify --disk-encryption-targets
+* Update Python SDK dependency to 9.0
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:65]: It is no longer required to add history notes here.
Instead, put it in PR title or add them to description of PR. Azure CLI will generate history notes automatically when releasing a new version.

**Comment by bgklein** [comment]
With the new azdev setup those tests will only record the old help text until the CLI is actually released. Don't think its really needed anymore as well.

**Comment by bgklein** [comment]
@qwordy Should be good for merge unless there are any new CLI steps we missed

**Comment by qwordy** [comment]
> 
> 
> @qwordy Should be good for merge unless there are any new CLI steps we missed

Done

**Pull request merged**